src/components/projects_home/projectsHome.tsx

```diff
@@ -1,8 +1,81 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ProjectSideBar from './projectsHomeSideBar';
+import ProjectCard from './projectsCard';
+import style from './projectHome.module.sass';
+import NavBar from '../navbar/navBar';
+import axios from 'axios';
+import BottomNavBar from '../navbar/bottomNavBar';
 
 function ProjectsList() {
+    const [projects, setProjects] = useState<any[]>([]);
+    const [contributors, setContributors] = useState<any[]>([]);
+    function retrieveProjects() {
+        axios.defaults.xsrfHeaderName = "X-CSRFTOKEN";
+        axios.defaults.xsrfCookieName = "csrftoken";
+        const token = window.localStorage.getItem('token')
+        axios.defaults.headers = {
+            'Content-Type': 'application/json',
+            Authorization: `Token ${token}`
+        }
+        let api = "http://127.0.0.1:8000/project/my-projects/";
+        axios.get(api).then(response => {
+            console.log(response);
+            setProjects(response.data);
+            setContributors(response.data.contributors)
+
+        })
+    }
+
+    let theme = window.localStorage.getItem('theme');
```

Comment:
Dark mode is still in beta. although it works some components return to being white after a page refresh (one so far)